CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
+For now, the team has set the following limits on pull requests:
+
+- Contributions beyond the level of a bug fix must be discussed with the team first, or they will likely be declined.
+- Pull requests must pass all Jenkins legs and be merge conflict free before they will be merged. 
+- Submissions must meet functional and performance expectations, including scenarios for which the team doesn’t yet have open source tests. This means you may be asked to fix and resubmit your pull request against a new open test case if it fails one of these tests.
+- Submissions must follow the [.editorconfig](http://editorconfig.org/) settings for each directory. 
+- Contributors must sign the [.NET CLA](https://cla2.dotnetfoundation.org/)
+
+When you are ready to proceed with making a change, get set up to [build](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md) the code and familiarize yourself with our developer workflow. 
```

Comment:
In reality this file almost never changes because it's confusing to everyone when it does. I don't think you can do dynamic links here, but I've also never seen it cause a problem. I'd almost go so far as to say if it does cause a problem, we're probably doing something wrong anyway. 😄